src/components/TableFilterListItem.js

```diff
@@ -1,9 +1,14 @@
 import Chip from '@material-ui/core/Chip';
 import PropTypes from 'prop-types';
 import React from 'react';
+import classnames from 'classnames';
```

Comment:
Would you accept a PR to replace `classnames` with [`clsx`](https://github.com/lukeed/clsx#usage)? Only suggesting because I learned today that [`@material-ui/core` uses `clsx`](https://material-ui.com/getting-started/faq/#whats-the-clsx-dependency-for).

Reply:
Just finished looking into it (https://github.com/mui-org/material-ui/pull/14152). Smaller, faster, same API, and Material UI switched from classnames to clsx. I'd would accept a PR for this change.